rules/windows/impact_backup_file_deletion.toml

```
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2021/10/01"
+maturity = "production"
+updated_date = "2021/10/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the deletion of Backup Files done by used other ways than using the backup solution.
+Adversaries may delete Backup files to ensure that recovery from a Ransomware attack is less likely.
```

```suggestion
description = """Identifies the deletion of backup files, saved using third-party software, by a process outside of the backup suite.
Adversaries may delete Backup files to ensure that recovery from a Ransomware attack is less likely or for the sake of data destruction.
```

Need to re-lint this (`python -m detection_rules toml-lint -f <file>`) or just manually break at 120 lines